app/controllers/herbaria/nexts_controller.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module Herbaria
+  # redirect to next/prev herbarium
+  class NextsController < ApplicationController
```

Comment:
is there a short, less awkward name for this controller?

Reply:
How about just enabling the herbaria#show action to parse next/prev params with a case statement? Or is that not possible. I feel like I've read through examples of simple CRUD controllers in the wild that handle next/previous, without another controller.